### PR TITLE
[Debugger] Fix debug hit failure for object init() method inside language and standard libraries

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will fix debug hit failure for object init() method inside language and standard libraries

Fixes https://github.com/ballerina-platform/ballerina-lang/issues/34258

### CODE REVIEW

File: tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/run/MultiModuleRunDebugTest.java [hunk 1]
```diff
@@ -124,6 +124,22 @@ public void testMultiModuleDebugScenarios() throws BallerinaTestException {
...
+                .resolve("any").resolve("modules").resolve("lang.query").resolve("types.bal");
+
+        debugTestRunner.addBreakPoint(new BallerinaTestDebugPoint(filePath, 84));
+        debugTestRunner.initDebugSession(DebugUtils.DebuggeeExecutionKind.RUN);
+
+        // Test for debug engage inside lang lib init() method
+        Pair<BallerinaTestDebugPoint, StoppedEventArguments> debugHitInfo = debugTestRunner.waitForDebugHit(25000);
+        Assert.assertEquals(debugHitInfo.getLeft().getSource(), debugTestRunner.testBreakpoints.get(0).getSource());
+        Assert.assertEquals(debugHitInfo.getLeft().getDAPBreakPoint().getLine(),
+                debugTestRunner.testBreakpoints.get(0).getDAPBreakPoint().getLine());
```
Comment (NipunaRanasinghe) [inline tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/run/MultiModuleRunDebugTest.java:140]: Assert.assertEquals(debugHitInfo.getLeft(), debugTestRunner.testBreakpoints.get(0));

Comment (praveennadarajah) [inline tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/run/MultiModuleRunDebugTest.java:140]: When adding breakpoints inside the lang libs, the file path URI schema for `debugHitInfo.getLeft()` is `bala:` and for `debugTestRunner.testBreakpoints.get(0)` is `file://`. Due this schema difference, we cannot do `Assert.assertEquals(debugHitInfo.getLeft()), debugTestRunner.testBreakpoints.get(0));`.

Therefore, asserting the source and line separately as shown above.

Comment (praveennadarajah) [reply tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/run/MultiModuleRunDebugTest.java:140]: Please refer https://github.com/ballerina-platform/ballerina-lang/pull/34515#discussion_r778699014

Comment (NipunaRanasinghe) [reply tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/run/MultiModuleRunDebugTest.java:140]: I see.. +1 for keeping it as it is

**Review by NipunaRanasinghe — APPROVED** [review]

**Comment by praveennadarajah** [comment]
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/34697

**Pull request closed without merging**